plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/cart-line-items-table/style.scss

```diff
@@ -163,7 +163,7 @@ table.wc-block-cart-items {
 			border-bottom: 1px solid $universal-border-light;
 
 			th {
-				padding: $gap-smaller $gap $gap 0;
+				padding: $gap-smaller $gap $gap-smaller * 1.25 0;
```

Comment:
`$gap-smaller * 1.25` is not something we have defined in the gap definitions. Maybe we should double check with @yjailin that this size (10px) is something we should be adding here. 

https://github.com/woocommerce/woocommerce-gateway-stripe/blob/6a7fafcd895c0a1253feee3fc4f7bf9d9c775f39/client/styles/abstracts/_variables.scss#L7